lib/repository.py

```diff
@@ -156,33 +162,35 @@ def run(self):
                 else:
                     script_version = None
 
-                if not file_exists(os.path.join(HOME_DIR, "scripts", script_name)):
-                    # File doesn't exist: download it
+                path_script_name = os.path.normpath(os.path.join(HOME_DIR, "scripts", script_name))
+ 
+                if not file_exists(path_script_name):
                     print "Downloading script: " + script_name
                     download_from_repository("scripts/" + script_name,
-                                             os.path.join(SCRIPT_WRITE_PATH, script_name))
-                elif script_version:
-                    # File exists: import and check MD5 sum
-                    need_to_download = False
+                                             os.path.normpath(os.path.join(SCRIPT_WRITE_PATH, script_name)))
+
+                # check MD5sum based on the script version to download the right scripts
+                # if MD5sum dont match; need_to_download is True
```

Comment:
"if MD5sum dont match; need_to_download is True" -> "if the MD5sum doesn't match need_to_download is set to True"